StratisCore.UI/src/app/setup/create/create.component.html

```diff
@@ -1,7 +1,8 @@
 <section id="GUI" class="container-fluid">
   <div class="row">
-    <div class="col-4 left-login d-flex align-items-center">
-        <img src="./assets/images/logo_white.svg" width="160px" height="auto">
+    <div class="col-4 left-login d-flex align-items-center" [class.left-login-green]="sidechainEnabled">
+      <img *ngIf="!sidechainEnabled" src="./assets/images/stratis/home-stratis-logo.svg" width="160px" height="auto">
```

Comment:
We could move this (two images with `ngIf`s) into a reusable component that takes `sidechainEnabled` as in input, that way if we ever change this we can do it in one place

Reply:
Currently going to leave this in as the logo change will happen when we load themes dynamically (soon TM)